### DISCUSSION TITLE
[FYI] Now available from ArchLinux User Repository (AUR)

### DISCUSSION THREAD

**Discussion by kseistrup** [MarshalX/atproto#470] [Show and tell]
By using an [AUR helper](https://wiki.archlinux.org/title/AUR_helpers) such as `yay`, ArchLinux users can now install the `atproto` module directly from [AUR](https://aur.archlinux.org/packages/python-atproto-git):

```sh
yay -S python-atproto
```

It will automatically pull in all dependencies, and install the module and the LICENSE along with the CHANGES, README, and SECURITY files.

[ You needn't do anything, and you may close this issue at your discretion . ]

**Comment by MarshalX** [comment]
thank you!
